core/src/main/java/org/aerogear/mobile/core/metrics/publisher/NetworkMetricsPublisher.java

```diff
@@ -53,23 +54,27 @@ public void publish(@NonNull final Metrics[] metrics, @Nullable final Callback c
 
             json.put("data", data);
 
-            httpRequest.post(url, json.toString().getBytes());
+            httpRequest.post(url, json.toString().getBytes())
```

Comment:
I think would be better to specify the charset to be used

Reply:
Let's make this a github issue and feature enhance it.  We will have to make the character set configurable which introduces more complexity.  This PR is mostly about porting the HTTP apis to a reactive pattern not add features to metrics.